setup.py

```diff
@@ -1,22 +1,5 @@
 #!/usr/bin/env python3
-import sys
 
-from setuptools import find_packages, setup
+from setuptools import setup
 
-needs_pytest = {"pytest", "test", "ptr"}.intersection(sys.argv)
-pytest_runner = ["pytest < 7", "pytest-runner"] if needs_pytest else []
-
-with open("README.md", encoding="utf-8") as fh:
-    long_description = fh.read()
-
-setup(
-    name="wdl2cwl",
-    url="https://github.com/common-workflow-lab/wdl-cwl-translator",
-    version="0.0.1",
-    author="Dinithi Wickramaratne",
-    author_email="diniwick124@gmail.com",
-    long_description_content_type="text/markdown",
-    packages=find_packages(),
-    setup_requires=[] + pytest_runner,
-    install_requires=["cwl-utils", "miniwdl", "regex"],
-)
+setup()
```

Comment:
a `setup.py` shouldn't be requited anymore with https://packaging.python.org/en/latest/tutorials/packaging-projects/#creating-pyproject-toml

Reply:
Ah! Today I Learned! Updated with a `pyproject.toml`, removing the `setup.py` (we can add it back later if we ever need some Python code in our build.)

Reply:
Updated `Makefile` as well. Not sure if the `pydocstyle` targets are correct though, but they passed locally for me.